src/Microsoft.Health.Fhir.Core/Features/KnownQueryParameterNames.cs

```diff
@@ -71,5 +71,7 @@ public static class KnownQueryParameterNames
         public const string OutputFormat = "_outputFormat";
 
         public const string TypeFilter = "_typeFilter";
+
+        public const string ExcludeLinks = "excludeLinks";
```

Comment:
Nit: How about put it in [EverythingParameterNames](https://github.com/microsoft/fhir-server/blob/main/src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/EverythingOperationParameterNames.cs)? This class maintains the unique parameters for $everything operation.

Reply:
And do we need to update the operation definition for patient-everything? Currently it points to FHIR spec in [OperationsCapabilityProvider](https://github.com/microsoft/fhir-server/blob/main/src/Microsoft.Health.Fhir.Shared.Api/Features/Operations/OperationsCapabilityProvider.cs), but the spec does not contain this input parameter.

Reply:
> Nit: How about put it in EverythingParameterNames? This class maintains the unique parameters for $everything operation.

Yes, good idea! Moved the const in 6a96e2a.

Reply:
> And do we need to update the operation definition for patient-everything? Currently it points to FHIR spec in [OperationsCapabilityProvider](https://github.com/microsoft/fhir-server/blob/main/src/Microsoft.Health.Fhir.Shared.Api/Features/Operations/OperationsCapabilityProvider.cs), but the spec does not contain this input parameter.

Really good question... I'm not sure. We didn't update the operation definition for export when we added the `_container` parameter (but we did document it [here](https://docs.microsoft.com/en-us/azure/healthcare-apis/data-transformation/export-data#query-parameters)). @CaitlinV39 thoughts?

Reply:
Created a user story to investigate updating the operation definitions for $export and patient $everything: [AB#85805](https://microsofthealth.visualstudio.com/Health/_workitems/edit/85805).